{{ cookiecutter.project_name }}/contribute.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "{{ cookiecutter.project_name }}",
+  "description": "{{ cookiecutter.project_description }}",
+  "repository": {
+    "url": "https://github.com/mozilla/{{ cookiecutter.repo_name }}/",
+    "license": "{{ cookiecutter.license }}",
+    "tests": "https://travis-ci.org/mozilla/{{ cookiecutter.repo_name }}/"
+  },
+  "participate": {
+    "home": "https://wiki.mozila.org/{{ cookiecutter.project_name|replace(' ', '_') }}",
+    "docs": "https://{{ cookiecutter.repo_name }}.readthedocs.org",
+    "mailing-list": "https://lists.mozilla.org/listinfo/dev-community-tools",
+    "irc": "irc://irc.mozilla.org/#{{ cookiecutter.repo_name }}-dev",
+    "irc-contacts": []
+  },
+  "bugs": {
+    "list": "https://github.com/mozilla/{{ cookiecutter.repo_name }}/issues",
+    "report": "https://github.com/mozilla/{{ cookiecutter.repo_name }}/issues/new"
+  },
+  "urls": {
+    "prod": "https://{{cookiecutter.repo_name }}.org",
+    "stage": "https://{{cookiecutter.repo_name }}.allizom.org",
+    "dev": "https://{{ cookiecutter.repo_name }}-dev.allizom.org"
+  },
+  "keywords": []
+}
```

Comment:
I'm concerned about setting up all these urls, wiki pages and bugzilla links automatically for the developer. Many mozilla projects don't have a unique bugzilla component, not all of them have wiki pages or dev, stage and prod instances following this pattern.

I'd only keep the `name`, `description`, `repository` keys and comment out or remove the rest to avoid ending up with a contribute.json which points to many 404s.